sc/aml-markers.tab.info

```diff
@@ -0,0 +1,24 @@
+{
+    "collection": "10x Genomics",
+    "description": "Cell type markers for bone marrow cells. Markers were manually extracted from the <a href='https://www.nature.com/articles/ncomms14049'>publication on bone marrow mononuclear cells with AML</a>.",
+    "instances": 8,
+    "name": "aml-markers",
+    "references": [
+        "Zheng GX, Terry JM, Belgrader P et al. (2017) <a href='https://www.nature.com/articles/ncomms14049'> Massively parallel digital transcriptional profiling of single cells, Nat Commun 8:14049.</a>"
+    ],
+    "seealso": [],
+    "size": 308,
+    "source": "<a href='https://www.nature.com/articles/ncomms14049'>10x Genomics</a>",
+    "tags": [
+        "aml",
+        "expression",
+        "markers",
+        "human"
+    ],
+    "target": "none",
```

Comment:
Correct JSON specification for this is:
`"target": null,`

(as is described in the README)